### ISSUE TITLE
inline iframes require title attributes (specifically with amp-brightcove) - a11y

### ISSUE THREAD

**Issue by randomock** [ampproject/amphtml#38530]
### Description

This is a contnuation/reopening of bug [#29874](https://github.com/ampproject/amphtml/issues/29874)

AMP components that create inline iframes do not have a title attribute. This is against [WCAG guidelines](https://www.w3.org/TR/WCAG20-TECHS/H64.html) and fails [Lighthouse audits](https://web.dev/frame-title/?utm_source=lighthouse&utm_medium=devtools).

It appears based on the comments in the bug mentioned above that this issue was resolved for amp-youtube elements but not resolved for amp-brightcove.  While running the [Access Assistant](https://chrome.google.com/webstore/detail/access-assistant/ojiighldhdmahfdnhfdebnpmlbiemdfm) extension to test accessibility on a page containing amp-brightcove elements, there remains an issue with the iframes containing the brightcove player not having a valid title attribute.

### Reproduction Steps

1. Visit https://s3-componentsexperiments-sbx01-cac1-01.s3.ca-central-1.amazonaws.com/AMP/index.html
2. Inspect the iframe that contains the brightcove player.  There is no title attribute.

Upon running Access Assistant on the above link, this issue comes up:
<img width="648" alt="Screen Shot 2022-11-04 at 12 07 36 PM" src="https://user-images.githubusercontent.com/68619065/200023804-4b081844-99e1-47ea-ad2a-41dfbbe1b4a0.png">

### Browser(s) Affected

Chrome, Firefox, Safari, Edge, UC Browser

### AMP Version Affected

2210211855000
